### DISCUSSION TITLE
Problem after  updating with  Purfview's Faster - Whisper

### DISCUSSION THREAD

**Discussion by dpassaggio90** [SubtitleEdit/subtitleedit#8046] [Q&A] [answered]
Yesterday after updating the latest version, the engine 
Purfiview’s Faster -Whisper no longer works properly.
The error that comes out is this:
"**Whisper SHA-512 hash does not match!** "
I tried to uninstall and reinstall the previous version but now it doesn’t work with the previous version either.
While on another pc where I haven’t updated, fortunately everything works normally.

I saw that in the new Update there is the CPP alternative that works correctly, but it is not as fast as Purfiview’s Faster -Whisper.

Is there any way to fix the problem? Can I copy some files from the other pc where everything works correctly?

**Comment by Purfview** [comment]
Dupe of https://github.com/SubtitleEdit/subtitleedit/issues/8042

**Comment by dpassaggio90** [comment, marked as answer]
Ok I took the file "Whisper-Faster_r186.1_windows" copied to the installation folder of Subtitle Edit, started, and after the prompt screen closes by itself, I restarted Subtitle Edit and it seems to work properly now, thanks for the solution!

**Comment by niksedk** [comment]
I've heard this could also be an anti-virus program issue.
